hgvs/decorators/lru_cache.py

```diff
@@ -32,6 +32,15 @@ def __init__(self, tup, hash=hash):
     def __hash__(self):
         return self.hashvalue
 
+    def __getstate__(self):
+        return True  # needs to be truthy for __setstate__ to be called
```

Comment:
This `__getstate__` and `__setstate__` setup prevents the `hashvalue` attribute from being pickled, which can lead to issues when hash randomization is enabled. The test suite is still unreliable with hash randomization enabled (not sure why), but this allowed more tests to pass. I think this is _technically_ unnecessary for this PR, but is an improvement I ended up implementing while trying & failing to make a py2/py3 compatible `cache.hdp` file.